gcpy/util.py

```diff
@@ -1705,8 +1717,12 @@ def get_filepath(
     date_str = date_str.replace("-", "")
     date_str = date_str.replace(":", "")
 
-    # Set file path to return
+    # Set file path. Include grid resolution if GCHP restart file.
     path = file_tmpl + date_str + extension
+    if is_gchp and "Restart" in collection and not gchp_is_pre_14.0:
+        date_str = date_time.replace("z", "")
+        path = file_tmpl + date_str + "00z.c" + gchp_res + extension
```

Comment:
If you wanted you could use an fstring:
f"{file_tmpl}{date_str}00z.c{gchp_res}{extension}"

I find them more readable, but up to you

Reply:
True, I'll leave that for someone down the road to fix up GCPy formatting for consistency.